macOS warnings

I guess this is based on the recent macOS basic types changes (@sonndinh @jrw972)

```
TAO/TAO_IDL/be/be_helper.cpp:374:20: warning: format specifies type 'unsigned long' but the argument has type 'ACE_CDR::ULongLong' (aka 'unsigned long long') [-Wformat]
```
There are a few instances of the same warning

Apple clang version 11.0.3 (clang-1103.0.32.62)
with -std=c++11

--- a/.github/workflows/macosx.yml
+++ b/.github/workflows/macosx.yml
@@ -10,12 +10,12 @@ jobs:
   build:
     strategy:
       matrix:
-        cxxstd: [3, 11]
+        cxxstd: ["03", "11"]
         include:
           - os: macos-10.15
             platform_file: include $(ACE_ROOT)/include/makeinclude/platform_macosx.GNU
     runs-on: ${{ matrix.os }}
-    name: "${{ matrix.os }}-${{ matrix.cxxstd }}"
+    name: "${{ matrix.os }}-C++${{ matrix.cxxstd }}"
     env:
       ACE_ROOT: ${{ github.workspace }}/ACE
       TAO_ROOT: ${{ github.workspace }}/TAO

--- a/.github/workflows/macosx.yml
+++ b/.github/workflows/macosx.yml
@@ -10,11 +10,12 @@ jobs:
   build:
     strategy:
       matrix:
+        cxxstd: [3, 11]
         include:
           - os: macos-10.15
             platform_file: include $(ACE_ROOT)/include/makeinclude/platform_macosx.GNU
     runs-on: ${{ matrix.os }}
-    name: ${{ matrix.os }}
+    name: "${{ matrix.os }}-${{ matrix.cxxstd }}"
     env:
       ACE_ROOT: ${{ github.workspace }}/ACE
       TAO_ROOT: ${{ github.workspace }}/TAO
@@ -31,9 +32,14 @@ jobs:
       run: |
         '#include "ace/config-macosx.h"' > ${env:ACE_ROOT}/ace/config.h
       shell: pwsh
+    - name: Add c++11 to platform_macros.GNU
+      if: matrix.cxxstd == 11
+      run: |
+        'c++11=1' > ${env:ACE_ROOT}/include/makeinclude/platform_macros.GNU
+      shell: pwsh
     - name: create $ACE_ROOT/include/makeinclude/platform_macros.GNU
       run: |
-        '${{ matrix.platform_file }}' >  ${env:ACE_ROOT}/include/makeinclude/platform_macros.GNU
+        '${{ matrix.platform_file }}' >> ${env:ACE_ROOT}/include/makeinclude/platform_macros.GNU
       shell: pwsh
     - name: create $ACE_ROOT/bin/MakeProjectCreator/config/default.features
       run: |

--- a/ACE/ace/config-macosx-lion.h
+++ b/ACE/ace/config-macosx-lion.h
@@ -10,11 +10,12 @@
 #if  defined (__x86_64__)
 // On 64 bit platforms, the "long" type is 64-bits.  Override the
 // default 32-bit platform-specific format specifiers appropriately.
-# define ACE_INT64_FORMAT_SPECIFIER_ASCII "%ld"
-# define ACE_INT64_FORMAT_SPECIFIER "%ld"
-# define ACE_UINT64_FORMAT_SPECIFIER_ASCII "%lu"
-# define ACE_UINT64_FORMAT_SPECIFIER "%lu"
-# define ACE_UINT64_FORMAT_SPECIFIER_ASCII "%lu"
+# if ! defined (ACE_HAS_CPP11)
+#   define ACE_INT64_FORMAT_SPECIFIER_ASCII "%ld"
+#   define ACE_INT64_FORMAT_SPECIFIER "%ld"
+#   define ACE_UINT64_FORMAT_SPECIFIER_ASCII "%lu"
+#   define ACE_UINT64_FORMAT_SPECIFIER "%lu"
+# endif /* ACE_HAS_CPP11 */
 # define ACE_SSIZE_T_FORMAT_SPECIFIER_ASCII "%ld"
 # define ACE_SIZE_T_FORMAT_SPECIFIER_ASCII "%lu"
 #endif /* __x86_64__ */

--- a/ACE/tests/Compiler_Features_32_Test.cpp
+++ b/ACE/tests/Compiler_Features_32_Test.cpp
@@ -45,7 +45,7 @@ A::u_type_::~u_type_ ()
 void A::clear ()
 {
 #if defined __clang__ && \
-    (defined __apple_build_version__ && __apple_build_version__ <= 11000033 \
+    (defined __apple_build_version__ && __apple_build_version__ <= 11030032 \
      || __clang_major__ <= 10)
 #define CLANG_WORKAROUND
 #endif

